Retroactive review followup: Fix API details and style nits in warm pools guide (#2430)
superfly/docs: blueprints/warm-pool-user-machines.html.md

    "network": "pool-x7f3k2"
  }'

# Allocate a Flycast (private) address into the app's own network
curl -X POST "https://api.machines.dev/v1/apps/pool-x7f3k2/ip_assignments" \
  -H "Authorization: Bearer ${FLY_API_TOKEN}" \
  -H "Content-Type: application/json" \
  -d '{"type": "private_v6", "network": "pool-x7f3k2"}'
```

Pass the same `network` name you created the app with. The `network` field on the allocation targets which 6PN network the Flycast address routes within, and it does not default to the app's own network. Leave it empty and the address lands in your organization's default network, where it cannot reach a Machine that lives in the app's isolated per-app network.

> App creation fails if the name is already taken. Your worker will retry after crashes, so treat creation as idempotent: on a name-collision error, `GET /v1/apps/{name}` to confirm the app exists and reuse it. If it doesn't exist, the failure was a real validation error, so rethrow it.

Reviewer: Optional: I reproduced the collision against the live API and it does return a specific code, `HTTP 422` with body `{"error":"Validation failed: Name has already been taken"}`. It's just absent from the response-code reference and OpenAPI spec (it's a Rails-style validation error). If we're comfortable citing an observed-but-undocumented code we can be specific here; if you'd rather stay conservative until it lands in the reference, your current wording is fine. Suggestion if we want to be specific:

```suggestion
App creation returns `422` if the name is taken. Your worker will retry after crashes, so treat that as idempotent: `GET /v1/apps/{name}` to confirm the app exists and reuse it. If it doesn't exist, the `422` was a real validation error, so rethrow it.
```

Author: Thanks for reproducing this! Since you've verified it against the live API, I'd rather be specific than vague, so I'm applying your suggestion. I've also added `422` to the Response Codes table in the Machines API reference.

    }
  }'

> # Wait for the VM to boot (timeout defaults to 60s)

Reviewer: I checked this against the live Machines API: 60s isn't only the default, it's the enforced maximum. `wait?...&timeout=61` returns `HTTP 400` with `invalid WaitMachineRequest.Timeout: value must be inside range [1s, 1m0s]` (and `timeout=60` succeeds). Worth noting, since a caller who wants to wait longer has to loop rather than raise the value.

```suggestion
# Wait for the VM to boot (timeout defaults to 60s, which is also the max)
```

Author: Thanks for checking that!
